Solutions/GoogleCloudPlatformDNS/Hunting Queries/GCPDNSRareErrors.yaml

```diff
@@ -0,0 +1,29 @@
+id: 9e3145ef-f6bf-4dd0-8783-a0847f68cb54
+name: Google DNS - Domains with rare errors
+description: |
+  'Query searches for Domains with rare errors.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: GCPDNSDataConnector
+    dataTypes:
+      - imDNSGCPCloudDNS
+tactics:
+  - CommandAndControl
+relevantTechniques:
+  - T1095
+query: |
+  imDNSGCPCloudDNS
+  | where TimeGenerated > ago(24h)
+  | where EventResultDetails != 'NOERROR'
+  | summarize count() by EventResultDetail, Query

```

Comment:
Since you are using the parser in queries, you will have to change field name EventResultDetail to EventResultDetails.
Only then the validations would pass. Thank You.

Reply:
fixed